ofl/hasubimono/METADATA.pb

```diff
@@ -0,0 +1,29 @@
+name: "hasubimono"
+designer: "Eli Heuer"
+license: "OFL"
+category: "MONOSPACE"
+date_added: "2023-09-13"
+fonts {
+  name: "hasubimono"
+  style: "normal"
+  weight: 400
+  filename: "Hasubi-Mono[wght].ttf"
```

Comment:
The file name should be HasubiMono[wght].ttf, without hyphen. It means you have to change you .glyphs file name.